### PR TITLE
[ios sdk] Remove support of webviews for iOS sdk

### PR DESCRIPTION
**Pull request by SerhiiBovtriuk**
Remove support of webviews and bump version
to 1.5.0

### CODE REVIEW

File: TDConnectIosSdk/Config.swift [hunk 1]
```diff
@@ -181,7 +173,6 @@ open class Config {
         self.accountId = accountId
         self.claims = claims
         self.optionalParams = optionalParams
-        self.isWebView = isWebView
```
Comment (ulfcarlin) [inline TDConnectIosSdk/Config.swift]: Should the actual variable be removed as well?

File: TDConnectIosSdk/OAuth2Module.swift [hunk 2]
```diff
@@ -375,7 +371,7 @@ open class OAuth2Module: NSObject, AuthzModule, SFSafariViewControllerDelegate {
                 self.handleCallback(successUrl, error: error, state: state, completionHandler: completionHandler)
             })
             (self.authenticationSession as! SFAuthenticationSession).start()
-        } else if browserType == .safariViewController {
+        } else if browserType == .safariViewController || browserType == .unknown {
```
Comment (ulfcarlin) [inline TDConnectIosSdk/OAuth2Module.swift]: Should unknown browser type result in external browser being used instead?

File: TDConnectIosSdk/Config.swift (hunk 1 above)
Comment (SerhiiBovtriuk) [reply TDConnectIosSdk/Config.swift]: Yes. This is a braking change and it should break the code to force people to change it and react, when they update.

File: TDConnectIosSdk/OAuth2Module.swift (hunk 2 above)
Comment (SerhiiBovtriuk) [reply TDConnectIosSdk/OAuth2Module.swift]: Will set defaulting to normal browser

**Pull request merged**